tests/unit/test_zowe_core.py

```diff
@@ -35,6 +39,88 @@ def keyring_get_password(serviceName: str, username: str):
 def keyring_get_password_exception():
     raise Exception
 
+matching_config_json = {
+    "$schema": "./zowe.schema.json",
+    "defaults": {
+        "zosmf": "zosmf",
+        "tso": "tso",
+        "ssh": "ssh",
+        "base": "base"
+    },
+}
+
+matching_schema_json = {
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "$version": "1.0",
+    "type": "object",
+    "description": "Zowe configuration",
+    "properties": {
+        "defaults": {
+            "type": "object",
+            "description": "Mapping of profile types to default profile names",
+            "properties": {
+                "zosmf": {
+                    "description": "Default zosmf profile",
+                    "type": "string"
+                },
+                "tso": {
+                    "description": "Default tso profile",
+                    "type": "string"
+                },
+                "ssh": {
+                    "description": "Default ssh profile",
+                    "type": "string"
+                },
+                "base": {
+                    "description": "Default base profile",
+                    "type": "string"
+                }
+            }
+        },
+    }
+}
+
+not_matching_config_json = {
+    "$schema": "./zowe.schema.json",
+    "defaults": {
+        "zosmf": "zosmf",
+        "tso": "tso",
+        "ssh": "ssh",
+        "base": "base"
+    },
+}
+
+not_matching_schema_json = {
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "$version": "1.0",
+    "type": "object",
+    "description": "Zowe configuration",
+    "properties": {
+        "defaults": {
+            "type": "object",
+            "description": "Mapping of profile types to default profile names",
+            "properties": {
+                "zosmf": {
+                    "description": "Default zosmf profile",
+                    "type": "boolean"
+                },
+                "tso": {
+                    "description": "Default tso profile",
+                    "type": "string"
+                },
+                "ssh": {
+                    "description": "Default ssh profile",
+                    "type": "string"
+                },
+                "base": {
+                    "description": "Default base profile",
+                    "type": "string"
+                }
+            }
+        },
+    }
+}
+
```

Comment:
I would move those definitions closer to the method they are used in or even better, define them in separate files and load them from the tests with `json.open(open("path"))`

Reply:
I'll refactor this as well.